<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>NetBR</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="main_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <!-- Top Box -->
    <item>
     <widget class="QWidget" name="top_box">
      <layout class="QVBoxLayout" name="top_layout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="cloud_icon"/>
       </item>
       <item>
        <widget class="QLabel" name="top_label">
         <property name="text">
          <string>Nuvem</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Spacer Top -->
    <item>
     <spacer name="spacer_top">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <!-- Button -->
    <item>
     <widget class="QPushButton" name="button">
      <property name="text">
       <string>TESTAR!</string>
      </property>
      <property name="minimumHeight">
       <number>80</number>
      </property>
     </widget>
    </item>
    <!-- Spacer Bottom -->
    <item>
     <spacer name="spacer_bottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <!-- Text Box -->
    <item>
     <widget class="QWidget" name="text_box">
      <layout class="QVBoxLayout" name="text_box_layout">
       <property name="leftMargin">
        <number>18</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scroll_area">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Progress Bar -->
    <item>
     <widget class="QProgressBar" name="progress_bar"/>
    </item>
    <!-- Bottom Box -->
    <item>
     <widget class="QWidget" name="bottom_box">
      <layout class="QVBoxLayout" name="bottom_layout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="mersen_logo"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
